vscode-dotnet-runtime-library/src/Utils/WebRequestWorker.ts

```diff
@@ -7,13 +7,26 @@ import axiosRetry from 'axios-retry';
 import { HttpsProxyAgent } from 'https-proxy-agent';
 import { getProxySettings } from 'get-proxy-settings';
 import { AxiosCacheInstance, buildMemoryStorage, setupCache } from 'axios-cache-interceptor';
-import {DiskIsFullError, DotnetDownloadFailure, EventBasedError, SuppressedAcquisitionError, WebRequestError, WebRequestSent } from '../EventStream/EventStreamEvents';
-import { getInstallFromContext } from './InstallIdUtilities';
-
+import * as dns from 'dns';
 import * as fs from 'fs';
 import { promisify } from 'util';
 import stream = require('stream');
+
```

Comment:
multiple empty lines?

Reply:
Thank you for your review. I wanted to separate the 2 components here based on my custom code and external code, but 1 line is good!